Orange/widgets/regression/owadaboostregression.py

```diff
@@ -0,0 +1,66 @@
+from PyQt4.QtCore import Qt
+
+from Orange.regression.base_regression import LearnerRegression
+from Orange.data import Table
+from Orange.regression import TreeRegressionLearner
+from Orange.ensembles import SklAdaBoostRegressionLearner
+from Orange.widgets import gui
+from Orange.widgets.classify import owadaboost
+from Orange.widgets.settings import Setting
+
+
+class OWAdaBoostRegression(owadaboost.OWAdaBoostClassification):
+    name = "AdaBoost"
+    description = "An AdaBoost regression algorithm."
+    icon = "icons/AdaBoost.svg"
+    priority = 80
+
+    LEARNER = SklAdaBoostRegressionLearner
+    OUTPUT_MODEL_NAME = "Model"
+
+    inputs = [("Model", LearnerRegression, "set_base_model")]
+    want_main_area = False
+    resizing_enabled = False
+
+    losses = ["linear", "square", "exponential"]
+
+    learner_name = Setting("AdaBoost Regression")
+    n_estimators = Setting(50)
+    learning_rate = Setting(1.)
+    loss = Setting(0)
+
+    def add_main_layout(self):
+        self.base_estimator = TreeRegressionLearner()
+        box = gui.widgetBox(self.controlArea, "Parameters")
```

Comment:
OWAdaBoostRegression is missing attribute self.base_label, which is used in parent's set_base_model method.